test/functional/test_site/testThumbnails.md

```diff
@@ -0,0 +1,90 @@
+# Thumbnail test
+
+## Images
+
+### Square thumb, size = 100 (default)
+<thumbnail src="http://placehold.it/800x600" />
+
+### Square thumb with alt="Test"
+<thumbnail alt="Test" src="asdf" />
+
+### Square thumb, size = 200
+<thumbnail src="http://placehold.it/800x600" size='200' />
+
+### Circle thumbs, size = 100
+<thumbnail src="http://placehold.it/800x600" circle />
+<thumbnail src="http://placehold.it/100x300" circle />
+
+### An actual image
+<thumbnail circle size=300 src="/images/deer.jpg" />
+<pic src="images/deer.jpg" height=300 />
```

Comment:
Seems like `<pic>` is giving us buddy behaviour within `testThumbnails.md`. 

Tested this line within `index.md` of `test_site` and it works. But here it just fails to load.